docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: explain
+- Status: draft
+- Start Date: 2015-08-10
+- RFC PR:
+- Cockroach Issue: #2047
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
```

Comment:
I'm confused by the naming. You keep referring to the statement as `EXPLAIN DEBUG`, but only `EXPLAIN` exists in postgres and mysql (with some options such as ANALYSE, VERBOSE, EXTENDED), and you never use `DEBUG` in your examples.